src/TimePicker/ClockHours.js

```diff
@@ -0,0 +1,197 @@
+// @flow weak
+
+import React, { PropTypes, Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import ClockNumber from './ClockNumber';
+import ClockPointer from './ClockPointer';
+import { getTouchEventOffsetValues, rad2deg } from '../utils/timeUtils';
+
+
+export const styleSheet = createStyleSheet('ClockHours', () => {
+  return {
+    root: {
+      height: '100%',
+      width: '100%',
+      borderRadius: '100%',
+      position: 'relative',
+      pointerEvents: 'none',
+      boxSizing: 'border-box',
+    },
+
+    hitMask: {
+      height: '100%',
+      width: '100%',
+      pointerEvents: 'auto',
+    },
+  };
+});
+
+class ClockHours extends Component {
+  static propTypes = {
+    format: PropTypes.oneOf(['ampm', '24hr']),
+    initialHours: PropTypes.number,
+    onChange: PropTypes.func,
+  };
+
+  static defaultProps = {
+    initialHours: new Date().getHours(),
+    onChange: () => {},
+    format: 'ampm',
+  };
+
+  static contextTypes = {
+    styleManager: PropTypes.object.isRequired,
+  };
+
+  static muiName = 'ClockHours';
```

Comment:
No need to add those `static muiName` if you don't need them. Hint: you don't. We use them to check children element type to apply custom behavior.